model/outgoing_webhook.go

```diff
@@ -51,6 +51,7 @@ type OutgoingWebhookResponse struct {
 	IconURL  string          `json:"icon_url"`
 	Props    StringInterface `json:"props"`
 	Type     string          `json:"type"`
+	PostType string          `json:"post_type"`
```

Comment:
This is probably my fault since I think I gave you `PostType` as a name :P but can we go with `ResponseType` so that it's less likely to be confused with the `post.Type` field?

Reply:
done